DistTestCore/Codex/CodexContainerRecipe.cs

```diff
@@ -22,7 +22,7 @@ public CodexContainerRecipe()
         {
             Image = GetDockerImage();
 
-            Resources.Requests = new ContainerResourceSet(milliCPUs: 1000, memory: 6.GB());
+            Resources.Requests = new ContainerResourceSet(milliCPUs: 100, memory: 100.MB());
```

Comment:
This are not realistic number under even the most optimised version of the software. Why are you pushing them so low?

Reply:
As we discussed, it is just a request - how much resources we do need on the node to be able to schedule a Codex Pod.
If the node has more resources, Codex will be able to use up to `4000/12.GB`, but if not it may get OOM or slow-down because of the CPU usage.

Maybe @benbierens can enlighten us on that, but for some Dists Tests we may run a lot of Pods, let say 20 and with the current values it will require `20 x 1vCPU/6GB = 20 vCPU/120GB`, even if resources will not be used.
And with the proposed settings `20 x 0.1vCPU/0.1GB = 2 vCPU/2GB`.

But yeah, it depends on exactly what we are doing during these tests and if we must guarantee some known resources value, we should adjust that.

I noted proposed values in OpenLens when we run Continuous Tests
<details>
<summary>Codex resources usage</summary>
<img width="1149" alt="Screenshot 2023-09-08 at 10 00 42" src="https://github.com/codex-storage/cs-codex-dist-tests/assets/20563034/5cb42a5e-5b75-4d40-b46c-71ed2287235d">
<img width="1145" alt="Screenshot 2023-09-08 at 10 00 54" src="https://github.com/codex-storage/cs-codex-dist-tests/assets/20563034/640eb9b2-10bb-4af6-964f-7df7f1bbde70">
</details>


And just an example when we set a very low value for resources request, but yes it is a Nginx and a different case :)
https://github.com/kubernetes/ingress-nginx/blob/fcda809ab034fffd14d8ff9b39441b2d12da420e/charts/ingress-nginx/values.yaml#L342-L348

Reply:
Thanks, this makes sense now!

Reply:
I propose we add an option for the test-writer to be able to decide if they want to reserve a lot of resources or not when they start a node. This could be more important in some tests than in others.